doc/configuration.md

```diff
@@ -37,6 +37,7 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
       * **format** (string): the filesystem format (ext4, btrfs, or xfs).
       * **_create_** (object): contains the set of options to be used when creating the filesystem. A non-null entry indicates that the filesystem shall be created.
+        * **_allowExisting_** (boolean): whether or not the create operation shall be skipped if the target filesystem is already formatted with the desired filesystem type.
```

Comment:
It'll probably be ok, but "force" is typically used as a "do this no matter what". Even though our version is used slightly differently. Actually you'd need to use them together if you wanted an X filesystem, overwriting existing if its Y or Z but not X.